src/translation_table.py

```diff
@@ -0,0 +1,267 @@
+from bs4 import BeautifulSoup
+
+
+EN = (
+    {
+        'variable': 'congressperson_name',
+        'name': 'Congressperson Name',
+        'desc': """Name used by the congress person during his term in
+            office. Usually it is composed by two elements: a given name and a
+            family name; two given names; or two forename, except if the head
+            of the Lower House explicitly alter this rule in order to avoid
```

Comment:
From what I understand: Câmara dos Deputados/Chamber of Deputies is the lower house, while Senado/Senate is the upper.